### DISCUSSION TITLE
Pausing keda autoscaling at cluster level

### DISCUSSION THREAD

**Discussion by rohit-gujral** [kedacore/keda#5956] [General]
Hi TEam,
I run around 30 apps in my cluster which has keda scaledobject enabled for autoscaling using cron, cpu scalers. 
I want to temporarily disable keda autoscaling, keeping HPA working as desired.
I know I can use 
```
metadata:
  annotations:
    autoscaling.keda.sh/paused-replicas: "value"
```
But I will have to do this for all 30 applications which is a pain. 
1. Isn't there is something at keda operator level to disable this ?
2. Our all apps use internal helm chart across all envs hence adding this setting temporarily can affect other clusters as well.
3. Also does adding the above annotation disables cpu based scaler as well ? which I don't want

**Comment by tomkerkhove** [comment]
I like this idea of cluster-wide control. Thoughts @kedacore/keda-maintainers?

**Reply by JorTurFer** [reply]
> Also does adding the above annotation disables cpu based scaler as well ? which I don't want

Yes, when you pause the autoscaling, KEDA pauses the autoscaling totally

**Reply by JorTurFer** [reply]
And yeah, I like the idea of pausing the autoscaling globally :)

**Reply by rohit-gujral** [reply]
HI @JorTurFer Is there any update on this ?

**Reply by JorTurFer** [reply]
There isn't any update on this, nobody has been interested on contributing to this idea yet

**Reply by rickbrouwer** [reply]
I think it would be good to make a feature request on the [issues](https://github.com/kedacore/keda/issues) page

**Reply by rickbrouwer** [reply]
Is it an idea to create an env variable in the keda-operator?
```
- name: KEDA_AUTOSCALING_PAUSED
  value: "true"
```

**Reply by JorTurFer** [reply]
Actually, scaling to 0 the operator will produce the same behavior and as both will mean a restart of the operator (to remove the env or to scale it from 0), I guess that just scaling to 0 is easier and doesn't require changes. I mean, IMHO the proposal should not restart the operator pod for anything during the process

**Reply by rickbrouwer** [reply]
Yeah, you're totally right. Let's wait for a feature request and see what a proper solution is for this use case.
